software/database/pom.xml

```diff
@@ -144,11 +144,24 @@
         </dependency>
 
         <dependency>
+            <groupId>${jclouds.groupId}.provider</groupId>
```

Comment:
Confirmed that we can delete the aws-ec2 dependency, but we do need to keep the `javax.ws.rs-api` dependency for the live tests to work (in eclipse at least). We should change it to use `${jax-rs-api.version}` though.